v20.2/authorization.md

```diff
@@ -5,40 +5,43 @@ toc: true
 redirect_from: [create-and-manage-users.html, roles.html]
 ---
 
-User authorization is the act of defining access policies for authenticated CockroachDB users. CockroachDB allows you to create, manage, and remove your cluster's SQL [users](#sql-users) and assign SQL-level [privileges](#assign-privileges) to the users. Additionally, you can use [role-based access management (RBAC)](#roles) for simplified user management.
+User authorization is the act of defining access policies for authenticated CockroachDB users. CockroachDB allows you to create, manage, and remove your cluster's SQL [users](#users-and-roles) and assign SQL-level [privileges](#assign-privileges) to them. Additionally, you can use [role-based access control (RBAC)](#roles) for simplified user management.
```

Comment:
This paragraph is very confused and probably inaccurate

- creating and removing users and roles are related to authenticatuon, not authorization
- the term "access policy" is related to both authentication and authorization, so it's confusing to see it here

What *is* related to authorization is the mapping of roles to SQL privileges, and the use of role membership to group users that share privileges. However, that idea is not even mentioned here! It should really be the first thing mentioned on this page.

The term "RBAC" was originally introduced inside documentation because of a product-level fad that the term would help drive sell—i.e. it is *not* a technical thing and does *not* have explanatory power. (The fact that no other database uses "RBAC" to structure its reference docs on authorization should be a strong hint.) We made a *mistake* to use it in the *reference* documentation - it should have been segregated to the "major release notes" and perhaps the main feature/comparison page. I think it's time to correct this mistake.

Reply:
Thanks for the helpful context and analysis.

I've updated the intro to convey the key points you suggested around roles and privileges.

I will include the other reorg (moving creation of users/roles) in a follow-up issue.